projects/1.18/assets/butchercraft/lang/zh_us.json

```
@@ -0,0 +1,201 @@
+{
+  "block.butchercraft.bbq_ribs_block": "炭烤排骨",
+  "block.butchercraft.beef_block": "生牛肉块",
+  "block.butchercraft.beef_wellington_block": "惠灵顿牛排",
+  "block.butchercraft.butcher_block": "案板",
+  "block.butchercraft.cooked_beef_block": "熟牛肉块",
+  "block.butchercraft.cooked_lamb_block": "熟羊肉块",
+  "block.butchercraft.cooked_pork_block": "熟猪肉块",
+  "block.butchercraft.cow_hide_carpet": "牛皮",
+  "block.butchercraft.drying_rack": "晾干架",
+  "block.butchercraft.haggis_block": "哈吉斯",
+  "block.butchercraft.hash_block": "肉丁土豆",
```

```suggestion
  "block.butchercraft.hash_block": "马铃薯杂烩",
```
```suggestion
  "block.butchercraft.hash_block": "土豆杂烩",
```
你可能需要 https://dict.mcmod.cn/

---

实际上“肉丁土豆”反而还是正确译名，这两个建议的更像是望文生义了